src/style.scss

```diff
@@ -98,47 +114,54 @@ body {
       }
     }
     div.details {
-      span.netmask {
-        display: inline-block;
-        width: 15%;
-      }
-      span.base {
-        display: inline-block;
-        width: 15%;
-      }
-      span.broadcast {
-        display: inline-block;
-        width: 15%;
+      display: inline-flex;
+      justify-content: space-around;
+      width: 100%;
+      @media (min-width: 640px) {
+        width: 640px;
       }
-      span.count {
-        display: inline-block;
-        width: 15%;
+      span {
+        display: block;
       }
-      span.first {
-        display: inline-block;
-        width: 15%;
+    }
+    div.break {
+      width: 100%;
+      height: 0px;
+      margin: 0px;
+    }
+    div.usable {
+      display: inline-flex;
+      justify-content: space-around;
+      margin-top: 30px;
+      width: 100%;
+      @media (min-width: 640px) {
+        width: 640px;
       }
-      span.last {
-        display: inline-block;
-        width: 15%;
+      span {
+        display: block;
       }
-      span.value {
-        font-size: modular-scale(1);
+    }
+    span.value {
+      font-size: 1.22em;
+      @media (max-width: $xl) {
+        font-size: calc(0.7em + 0.6vw);
       }
-      span.label {
-        display: block;
-        font-family: 'Ubuntu', sans-serif;
-        text-transform: uppercase;
-        font-size: modular-scale(2);
-        font-weight: bold;
-        color: #656582;
+    }
+    span.label {
+      display: block;
+      font-family: 'Ubuntu', sans-serif;
+      text-transform: uppercase;
+      font-size: 1.22em;
+      @media (max-width: $xl) {
+        font-size: calc(0.7em + 0.6vw);
       }
+      font-weight: bold;
+      color: #656582;
     }
   }
 
   footer {
```

Comment:
Move the footer in tighter since it really corresponds strongly to the last row of flex data.